### PR TITLE
YOPN-520: Ping Pong

### LINKED ISSUE #1708: Paragraph Backporting: Ping Pong Block + Feature Card
As a Y content admin, I want to use the Ping Pong Paragraph and Feature Cards Paragraph that is being used by other Y associations powered by Image X so that I can better communicate my marketing materials online.

### PR DESCRIPTION
**Pull request by vitalii-stepaniuk**
Adding Ping Pong paragraph

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_ping-pong.scss [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+  &.theme-light {
+    .ping-pong-content-title,
+    .ping-pong-content-description,
+    p {
+      color: $gray-400;
+    }
+
+    &.defailt-bg {
+      background-color:#faf5ff;
```
Comment (anpolimus) [inline themes/openy_themes/openy_carnation/src/scss/paragraphs/_ping-pong.scss]: Please, reuse varible here.

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_ping-pong.scss [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+  &.theme-dark {
+    .ping-pong-content-title,
+    .ping-pong-content-description,
+    p {
+      color: $white;
+    }
+
+    &.default-bg {
+      background-color: #12619e;
```
Comment (anpolimus) [inline themes/openy_themes/openy_carnation/src/scss/paragraphs/_ping-pong.scss]: var

**Review by anpolimus — CHANGES_REQUESTED** [review]
TODO: Styling exist only for Carnation.
Please, add Rose and Lily

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
Outdated and we decided to go with [Paragraph Skins](https://github.com/ymcatwincities/paragraph_skins) when there is a need for current components alteration

**Pull request closed without merging**